src/components/forms/product-form/product-form.js

```diff
@@ -326,141 +333,156 @@ const ProductForm = ({ isEdit }) => {
                 handleChange={handleChange}
                 handleBlur={handleBlur}
                 handleSubmit={handleSubmit}
-                setFieldValue={setFieldValue}
                 toggleFieldsChanged={toggleFieldsChanged}
+                setFieldValue={setFieldValue}
               />
-            </Box>
-            <Box mt={3}>
-              <Divider />
-            </Box>
-            <Box mt={2}>
-              <Typography className={styles.title}>{PRODUCT_PRICE}</Typography>
-            </Box>
-            <Box mt={3} ml={1}>
-              <TextField
-                className={styles.input}
-                label={`${priceLabel.label}*`}
-                type={productFormValues.number}
-                name={priceLabel.name}
-                inputProps={{ min: 0 }}
-                error={touched[priceLabel.name] && !!errors[priceLabel.name]}
-                value={values[priceLabel.name]}
-                onChange={handleChange}
-                onBlur={handleBlur}
-              />
-            </Box>
-          </Paper>
-        </Grid>
-      </Grid>
-      {showCommentsPanel()}
+            </Grid>
+            <Grid item xs={12}>
+              <Paper className={styles.paper}>
+                <Box mb={1}>
+                  <Typography className={styles.title}>
+                    {PRODUCT_SPECIFICATION}
+                  </Typography>
+                </Box>
+                <ProductSpeciesContainer
+                  models={models}
+                  patterns={patterns}
+                  categories={categories}
+                  closures={closures}
+                  sizes={sizes}
+                  setSizes={setSizes}
+                  values={values}
+                  errors={errors}
+                  touched={touched}
+                  handleChange={handleChange}
+                  handleBlur={handleBlur}
+                  handleSubmit={handleSubmit}
+                  setFieldValue={setFieldValue}
+                  toggleFieldsChanged={toggleFieldsChanged}
+                />
+                <Box mt={3}>
+                  <Divider />
+                </Box>
+                <Box mt={2}>
+                  <Typography className={styles.title}>
+                    {PRODUCT_MATERIALS}
+                  </Typography>
+                </Box>
+                <Box mt={3} ml={1}>
+                  <ProductMaterialsContainer
+                    innerMaterials={materials?.inner || []}
+                    innerColors={innerColors}
+                    mainMaterials={materials?.main || []}
+                    mainColors={mainColors}
+                    bottomMaterials={materials?.bottom || []}
+                    bottomColors={bottomColors}
+                    values={values}
+                    errors={errors}
+                    touched={touched}
+                    handleChange={handleChange}
+                    handleBlur={handleBlur}
+                    handleSubmit={handleSubmit}
```

Comment:
u can use 
`const myFormProps = {values, errors, touched, handleChange, handleBlur, handleSubmit};
// and then in render
{...myFormProps}`

instead of 
`values={values}
errors={errors}
touched={touched}
handleChange={handleChange}
handleBlur={handleBlur}
handleSubmit={handleSubmit}`